src/computer.h

```diff
@@ -41,7 +41,6 @@ enum computer_action {
     COMPACT_BLOOD_ANAL,
     COMPACT_DATA_ANAL,
     COMPACT_DISCONNECT,
-    COMPACT_BONESETTING,
```

Comment:
Will this not break the logic of existing  computers in savegames?

Reply:
Tested right now. The game don't crash, no errors popping up. Existing medical computers won't be working, of course, activating them will show "evacuation" message. I think that's acceptable.